barretenberg/acir_tests/flows/prove_then_verify_tube.sh

```diff
@@ -5,7 +5,7 @@ mkdir -p ./proofs
 
 VFLAG=${VERBOSE:+-v}
 
-$BIN prove --scheme client_ivc --input_type compiletime_stack $VFLAG -c $CRS_PATH -b ./target/program.json
+$BIN write_arbitrary_valid_proof_and_vk_to_file --scheme client_ivc $VFLAG -c $CRS_PATH
```

Comment:
We can no longer generate a full ClientIvc proof from existing noir programs so I just added a flow that produces a valid proof/vk and writes them to file.